Homework_4/hw4.py

```diff
@@ -0,0 +1,94 @@
+user = {
+    'PIN': 9876,
+    'balance': 100
+}
+
+
+
+for i in range(3):
+    PIN = int(input('PROVIDE PIN: '))
+    try:
+        if len(str(PIN)) !=4:
+            raise Exception
+    except:
+        print('PIN must have 4 digits')
+    else:
+        if PIN != user['PIN']:
+            print('incorrect PIN')
```

Comment:
Instead of printing 'incorrect PIN', it is better to raise an exception here i.e. custom exception
```python
raise UnauthorisedError
```